Tests/SourceKitLSPTests/DocumentTestDiscoveryTests.swift

```diff
@@ -1574,4 +1660,6 @@ final class DocumentTestDiscoveryTests: XCTestCase {
     let tests = try await project.testClient.send(DocumentTestsRequest(textDocument: TextDocumentIdentifier(uri)))
     XCTAssertEqual(tests, [])
   }
+
+  func testDummy() {}
```

Comment:
nit: leftover from debugging?

Reply:
Thanks for spotting. Was a leftover to work around https://github.com/swiftlang/vscode-swift/issues/1183.